Dockerfile

```diff
@@ -1,8 +1,7 @@
-FROM python:3.11.11-slim-bullseye
-COPY scripts/requirements.txt .
-RUN pip install -r requirements.txt
+FROM pytorch/pytorch:2.5.1-cuda12.4-cudnn9-runtime
 COPY . /app
 WORKDIR /app
 RUN pip install .
+RUN pip install jupyter
 
-CMD ["polygraph_server"]
+CMD ["/bin/bash"]
```

Comment:
Not sure which command to launch by default, so I made /bin/bash (let the user decide)